java-frontend/src/main/java/org/sonar/plugins/java/api/tree/CaseLabelTree.java

```diff
@@ -40,9 +40,27 @@
 
   SyntaxToken caseOrDefaultKeyword();
 
+  /**
+   * @deprecated use the {@link #expressions()} method.
+   */
+  @Deprecated
```

Comment:
Add `@since 5.12` tag for deprecated methods, in order to know from when we introduce the change.

Reply:
Ok, I will add: `/** @deprecated (since 5.12) use ...`
To not confuse with `@since 5.12` that indicate the creation date of the method.